test/WebSites/ResponseCacheWebSite/Controllers/ClassLevelCacheController.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace ResponseCacheWebSite
+{
+    [ResponseCache(Duration = 100, Location = ResponseCacheLocation.Any, VaryByHeader = "Accept")]
+    public class ClassLevelCacheController
+    {
+        public string GetHelloWorld()
+        {
+            return "Hello, World!";
+        }
+
+        public string GetFooBar()
+        {
+            return "Foo Bar!";
+        }
+
+        [ResponseCache(Duration = 20)]
+        public string ConflictExistingHeader()
```

Comment:
Could you add one more where it says `NoStore` in the action? Might reflect a more realistic scenario where you want to cache everything in a specific controller except an action.